lib/hyperglot/data/squ.yaml

```diff
@@ -0,0 +1,20 @@
+name: Squamish
+orthographies:
+- autonym: Sḵwx̱wú7mesh sníchim
+  base: A Á C E É H I Í K Ḵ L M N P S T U Ú W X Y a á c e é h i í k ḵ l m n p s t u ú w x y 7 ʼ ’ ''
+  marks: ◌̓ ◌̱
+  punctuation: . , - ? !
+  script: Latin
+  status: primary
+  design_requirements:
+  - Curly comma accents and right quotes are preferred (used on consonants), to differentiate from acute accents (used on vowels).
```

Comment:
Keep the note, but those consonants should be in `base` as they are spelled, non? You can simply add `p̓ t’ ts̓ ...`

And as per my comment in the issue, feel free to add digraphs! 👌

Reply:
Digraphs and base+mark pairs are added now.